### PR TITLE
PT-3580: Lookup fields in advanced filters and export/push dialog need to have a uniform look

### PR DESCRIPTION
**Pull request by veronikaslc**
- Added user icon in the beginning of the input field

### CODE REVIEW

File: components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.css [hunk 1]
```diff
@@ -125,3 +125,9 @@
 .filter-match-count {
   margin-left: 0.5em
 }
+input.suggestUsersAndGroups {
+  background-image: url(/resources/icons/datamodel/users.png);
```
Comment (sdumitriu) [inline components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.css]: This doesn't work when PhenoTips is not deployed as the root webapp. Use `$xwiki.getSkinFile('icons/datamodel/users.png')`

File: components/widgets/ui/src/main/resources/XWiki/DateTimePicker.xml [hunk 2]
```diff
@@ -2890,7 +2890,7 @@ var SimpleDateFormat;
       <code>#template('colorThemeInit.vm')
 input.xwiki-date {
   background-image: url("$xwiki.getSkinFile('icons/silk/date.png')");
-  background-position: right center;
+  background-position: .3em center;
```
Comment (sdumitriu) [inline components/widgets/ui/src/main/resources/XWiki/DateTimePicker.xml]: Once you enter a date, the date is overlapping the icon and thus is unreadable. Also add a `2em` left padding.

File: components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.css [hunk 3]
```diff
@@ -125,3 +125,9 @@
 .filter-match-count {
   margin-left: 0.5em
 }
+input.suggestUsersAndGroups {
```
Comment (sdumitriu) [inline components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.css]: This doesn't really belong here, move it to `PhenoTips.Widgets`, in the SSX object named `Suggest workgroups`, which should be renamed to `Suggest users, groups and workgroups`.

**Comment by veronikaslc** [comment]
@sdumitriu codereview comments addressed

**Pull request merged**